Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -135,8 +135,7 @@ namespace AzToolsFramework
 
         const float viewAlignedScale = 1.12f;
         m_viewAngularManipulator->SetView(CreateManipulatorViewCircle(
-            *m_viewAngularManipulator, AZ::Color(1.0f, 1.0f, 1.0f, 1.0f), radius * viewAlignedScale, m_circleBoundWidth,
-            DrawFullCircle));
+            *m_viewAngularManipulator, AZ::Color(1.0f, 1.0f, 1.0f, 1.0f), radius * viewAlignedScale, m_circleBoundWidth, DrawFullCircle));
```

Comment:
nit: You can take advantage of already created colors: `AZ::Colors::White`.

Reply:
Good call, I can update this 👍